tutorial/README.md

```diff
@@ -0,0 +1,120 @@
+one of your own images, but for the purposes of this tutorial, we will use a multiplexed, high-resolution 
+[Perkin Elmer](https://downloads.openmicroscopy.org/images/Vectra-QPTIFF/perkinelmer/) (1.95 GB) image made available 
+under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) on OME.
+
+```bash
+$ wget https://downloads.openmicroscopy.org/images/Vectra-QPTIFF/perkinelmer/PKI_scans/LuCa-7color_Scan1.qptiff 
+```
+
+After the image has finished downloading, there are two options for creating an Avivator/Viv-compliant image. 
+
+
+### Pyramid Generation
+
+#### Option 1: Create a Bio-Formats "raw" Zarr
+
+The first option is to use `bioformats2raw` with `--file_type=zarr`. The default "raw" file type is currently 
+[n5](https://github.com/saalfeldlab/n5), so the flag is required to generate the zarr-based output. This command will
+create the OME-XML metadata along with a pyramidal Zarr for high-resolution images.
+
+```bash
+$ bioformats2raw LuCa-7color_Scan1.qptiff LuCa-7color_Scan1/ --file_type=zarr
+```
+
+`bioformats2raw` creates the file directory `LuCa-7color_Scan1/` which contains the "raw" bioformats output. The root directory 
+contains a `METADATA.ome.xml` file along with a `data.zarr/` directory containing the zarr
+output. This output can be viewed directly with [Avivator] by serving the top-level directory (`LuCa-7color_Scan1/`) 
+over HTTP ([see below](#viewing-in-avivator)).
+
+
+#### Option 2: Create an OME-TIFF
+
+The second option is to run the complete Bio-Formats pipeline to generate a valid OME-TIFF. 
+
+```bash
+$ bioformats2raw LuCa-7color_Scan1.qptiff n5_tile_directory/ 
+$ raw2ometiff n5_tile_directory/ LuCa-7color_Scan1.ome.tif --compression=zlib
+```
+
+> NOTE: Viv currently uses [`geotiff.js`](https://geotiffjs.github.io/) for accessing data from remote TIFFs 
+> over HTTP. There is limited compression support in this library, so the `--compression` flag must be specified with
+> either `zlib` or `Uncompressed` for viewing in the browser.
+
+
+### Viewing in Avivator
+
+Avivator is a purely client-side application and relies on access to data over HTTP. Therefore, viewing a dataset locally 
+requires a simple web-server. The easist option is to use [`http-server`](https://github.com/http-party/http-server#readme),
+which can be installed via `npm` or `Homebrew` if using a Mac:
+
+#### Install `http-server`
+
+```bash
+$ npm install --global http-server
+# or 
+$ brew install http-server
+```
+
+
+#### Starting a Server
+
+From within this directory, start a local server and open [Avivator] in your browser. 
+
+```bash
+$ http-server --cors='*' --port 8000 .
+```
+
+This command starts a web-server and makes the content in the current directory readable over HTTP. Once the server is running,
+open [Avivator] and paste `http://localhost:8000/LuCa-7color_Scan1/` (zarr) or `http://localhost:8000/LuCa-7color_Scan1.ome.tif` (OME-TIFF) 
```

Comment:
```suggestion
open [Avivator] and paste [`http://localhost:8000/LuCa-7color_Scan1/`](http://avivator.gehlenborglab.org/?image_url=http://localhost:8000/LuCa-7color_Scan1/) (zarr) or [`http://localhost:8000/LuCa-7color_Scan1.ome.tif`](http://avivator.gehlenborglab.org/?image_url=http://localhost:8000/LuCa-7color_Scan1.ome.tif) (OME-TIFF) 
```

You could even add links with the `image_url` parameter pre-filled for convenience

Reply:
Although this may make copying and pasting more difficult...